zipfs/fs.go

```diff
@@ -0,0 +1,111 @@
+package zipfs
+
+import (
+	"archive/zip"
+	"os"
+	"path/filepath"
+	"syscall"
+	"time"
+
+	"github.com/spf13/afero"
+)
+
+type Fs struct {
+	r     *zip.Reader
+	files map[string]map[string]*zip.File
+}
+
+func splitpath(name string) (dir, file string) {
```

Comment:
This is used in a couple of places and since it is a self-contained helper, it would be nice to add a couple of unit tests for it.

Reply:
From my point of view this might as well happen post-merge. If this PR ever gets merged, that is.